src/client/styles/objects/_reading-width.scss

```diff
@@ -1,3 +1,4 @@
 .reading-width {
   @include reading-width;
+  max-width: none !important;
```

Comment:
This attribute is set within the mixin on the line above to ensure line length is the [correct measure for readability](https://www.smashingmagazine.com/2014/09/balancing-line-length-font-size-responsive-web-design/) so it might be better just to remove the mixin include. 

If the reason for this change is that the restriction of an element that has this class or sits within an element that has this class is causing a problem with layout though it might be better to look at the HTML structure first rather than amending this particular class.

It's also good practice not to use `!important` but to use the cascade correctly to override properties.